packager/transformer.js

```diff
@@ -43,7 +43,7 @@ const getBabelRC = (function() {
     // (which works because we pass in `filename` and `sourceFilename` to
     // Babel when we transform).
     let projectBabelRCPath;
-    if (projectRoots && projectRoots.length > 0) {
+    if (!ignoreBabelRC && projectRoots && projectRoots.length > 0) {
```

Comment:
@satya164 what do you think aboutexposing `getBabelRC` on `rn-config.cli.js`. Now that I see this implemented this way I feel like it would have more semantic e to have this line (`!ignoreBabelRC && projectRoots && projectRoots.length > 0` inside the cli configuration.

cc @skevy